lib/main/main.rb

```diff
@@ -285,6 +285,7 @@
       end
       accept_thread = Thread.new { $_CLIENT_ = SynchronizedSocket.new(listener.accept) }
       localport = listener.addr[1]
+      Frontend.create_session_file(Account.character, listener.addr[2], listener.addr[1])
```

Comment:
This creates a session file for official FE - I don't think it's useful there.  And this exists in the detachable client, where I think it needs to live if the client can utilize it.  Suggest dropping this one, and updating the one in detachable client to use `Account.character`.

Reply:
Ok, to resolve I'm going to add a `front-end.rb` to permit apps / FEs / scripts to find the session file location for the current character.  And I'm going to modify this one to remove the now duplicative session file creation under detachable client.  That way they'll be ready for review.